elfloader-tool/src/arch-arm/sys_boot.c

```diff
@@ -23,7 +23,7 @@
 #define DTB_MAGIC (0xedfe0dd0)
 
 /* Maximum alignment we need to preserve when relocating (64K) */
-#define MAX_ALIGN_BITS (14)
+#define MAX_ALIGN_BITS (16)
```

Comment:
It would be helpful  to have this parts of the commit comment as comment in the sources as a reference
/* The 64 kiB alignment is a maximum requirement for a stage2
 * concatenated pagetable. See Table G5-4 in ARM DDI 0487I.a, page
 * G5-9186.
 */